src/unix/stream.c

```diff
@@ -533,6 +533,53 @@ void uv__server_io(uv_loop_t* loop, uv__io_t* w, unsigned int events) {
 }
 
 
+static void uv__process_queued_fds(uv_stream_t* server, int err) {
```

Comment:
I think it looks weird that this function takes that previous error. You may only wany yo re-arm the io handle in uv_accept, since you know it will go ok in the reject case.

Reply:
Not quite sure about this one. Whether the `uv_accept` fails or not; libuv will dequeue in advance for the next `uv_accept` call or re-arm the io handle if everything has been read.

We don't have any guarantee that every socket that passes thru' `uv_reject` is not the last one.